.circleci/images/docker-compose.yml

```diff
@@ -0,0 +1,71 @@
+        echo '["kni://b299bf5bb0b5e91ddf35e055bef9b69c38112bae582b589d4ab4955f6b0f8e790a182530bd57cff00cad1eb49c39d6ab66eaaa979c5d644681ee124649c5f00f@172.16.239.10:32323?discport=0\u0026ntype=cn"]' > /klaytn/static-nodes.json
+        kcn --datadir "/klaytn" init "/klaytn/genesis.json"
+        kcn \
+        --identity "CN-0" \
+        --rpc \
+        --rpcaddr "0.0.0.0" \
+        --rpcport "8551" \
+        --rpccorsdomain "*" \
+        --datadir "/klaytn" \
+        --port "32323" \
+        --rpcapi "db,klay,net,web3,miner,personal,txpool,debug,admin,istanbul" \
+        --networkid "2018" \
+        --nat "any" \
+        --nodekeyhex "871ccee7755bb4247e783110cafa6437f9f593a1eaeebe0efcc1b0852282c3e5" \
+        --nodiscover \
+        --debug \
+        --metrics \
+        --prometheus \
+        --syncmode "full" \
+        --rewardbase 0x73718c4980728857f3aa5148e9d1b471efa3a7dd
+
+    networks:
+      app_net:
+        ipv4_address: 172.16.239.10
+    restart: "no"
+  prometheus:
```

Comment:
I don't think we really need to run prometheus for CI. Please remove this section and the file `prometheus.yml`.

Reply:
I think so!! Thanks for letting me know.